src/world/Network/Handlers/GMCommandHandlers.cpp

```diff
@@ -610,12 +622,26 @@ void Sapphire::Network::GameConnection::gm2Handler( FrameworkPtr pFw,
     }
     case GmCommand::Jump:
     {
+      player.prepareZoning( targetPlayer->getZoneId(), true, 1, 0 );
+      if( player.getCurrentInstance() )
+      {
+        player.exitInstance();
+      }
+      if( targetPlayer->getCurrentInstance() && player.getCurrentInstance() != targetPlayer->getCurrentInstance() )
+      {
+        auto instance = targetPlayer->getCurrentInstance();
+        auto pInstanceContent = instance->getAsInstanceContent();
+        // Not sure if GMs actually get bound to an instance they jump to on retail. It's mostly here to avoid a crash for now.
+        pInstanceContent->bindPlayer( player.getId() );
+        player.setInstance( targetPlayer->getCurrentInstance() );
+      }
       if( targetPlayer->getZoneId() != player.getZoneId() )
```

Comment:
this won't work properly for housing zones (interior and exterior areas) because the zone ids are the same.

What you should do here is get the current zone and then check the guids and then move the player accordingly, that should then be portable across every territory type unless we add something similar to instance binding to another territory type